modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```diff
@@ -548,6 +562,61 @@ Resources:
                       "log_stream_name": "default_logs_{instance_id}"
                     }
                   }
+            'c:\cfn\scripts\AmazonCloudWatchAgent\install-python.ps1':
```

Comment:
Python is installed as part of the Watchmaker install here:  https://github.com/plus3it/terraform-aws-watchmaker/blob/e5e5b28641907163de203f1039eb9cc2baae3768/modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml#L620

Any reason why you're installing it again?

Reply:
Converted the windows cwl config manipulation to powershell and removed this quirk